.github/workflows/python-package.yml

```diff
@@ -12,6 +12,8 @@ jobs:
     strategy:
       matrix:
         python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
+      # Limit parallel runs on pushes to master to 1 to limit integration test rate limited failures
+      max-parallel: ${{ github.event_name == 'pull_request' && 5 || 1 }}
```

Comment:
This fake ternary syntax is shown in the [github examples](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#example)